app/models/BannerDesign.scala

```diff
@@ -124,4 +132,5 @@ case class BannerDesign(
   headerImage: Option[HeaderImage],
   colours: BannerDesignColours,
   lockStatus: Option[LockStatus],
+  fonts: Option[Font],
```

Comment:
A better way to match the typescript model would be:
```
sealed trait FontSize
object FontSize {
  case object small extends FontSize
  case object medium extends FontSize
  case object large extends FontSize

  implicit val decoder: Decoder[FontSize] = deriveEnumerationDecoder[FontSize]
  implicit val encoder: Encoder[FontSize] = deriveEnumerationEncoder[FontSize]
}
case class Font(size: FontSize)
case class Fonts(heading: Font)
```

And then:
`fonts: Option[Fonts]`